Automattic/sensei
# Prepare grading to support co-teachers.

Fixes 1903-gh-Automattic/sensei-pro

### Changes proposed in this Pull Request
* Add `sensei_grading_allowed_user_ids` hook to grant access to grading activity to additional users a part from the course author.
* Fixing some codestyle.
* Fixing wrong cache key for counters calculation.

### Testing instructions
- Confirm tests pass.
- Confirm that from a security perspective teachers (or any other non-admin user) gets unauthorised access to grading.

### New/Updated Hooks
* New `sensei_grading_allowed_user_ids` hook.
